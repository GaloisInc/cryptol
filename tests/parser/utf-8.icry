:l utf-8.cry
:t δοκιμή
:t テスト
