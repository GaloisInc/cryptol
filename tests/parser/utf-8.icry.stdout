Loading module Cryptol
Loading module Cryptol
Loading module Main
δοκιμή : [8]
テスト : [8]
